src/components.d.ts

```diff
@@ -244,17 +244,29 @@ export namespace Components {
         /**
           * Array of items filtered by titleQuery used in la-toc-item to determine with item is shown or not
          */
-        "filteredItems": Set<TOCItem> | null;
+        "filteredItems": Set<TOCItem1> | null;
```

Comment:
for some reason the build is generating this
```
import { TOCItem } from "./components/table-of-contents/table-of-contents";
import { TOCItem as TOCItem1 } from "./components/table-of-contents/table-of-contents";
```

and `TOCItem1` is being used in the interface `LaTableOfContentsController`
It doesnt break anything. Its just really odd..
Havent found any issues related to this build behavior in stencil's issues